src/Workspaces/CSharp/Portable/Formatting/Rules/SuppressFormattingRule.cs

```diff
@@ -124,9 +124,11 @@ private void AddSpecificNodesSuppressOperations(List<SuppressOperation> list, Sy
                 }
             }
 
-            if (node is AnonymousMethodExpressionSyntax anonymousMethod)
+            if (node is AnonymousFunctionExpressionSyntax ||
+                node is LocalFunctionStatementSyntax)
             {
-                AddSuppressWrappingIfOnSingleLineOperation(list, anonymousMethod.DelegateKeyword, anonymousMethod.GetLastToken(includeZeroWidth: true));
+                AddSuppressWrappingIfOnSingleLineOperation(
```

Comment:
you can put that IgnoreElastic option here. I am not sure why default rule would make some of construct to ignore elastic trivia, but looks like we already have some of those. so probably fine.

Reply:
http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/Formatting/Rules/ElasticTriviaFormattingRule.cs,46

Reply:
What about https://github.com/dotnet/roslyn/pull/24867 as the potential solution?

Reply:
that will work but, have you tried just adding IgnoreElastic option to the rule you added? that feels more simple. and looks like 
http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Formatting/Rules/Operations/SuppressOption.cs,2a0df9bb73a427bd,references

we already have such rules.

Reply:
okay, let me revise my answer. if you think this local single line case is specific to this code fix, then +1 on what you did (removing all elastic trivia)

if you think, it is better behavior in general, I think you should add that case in ElasticTriviaFormattingRule with IgnoreElastic option.

Reply:
Yeah, i think the behavior is better in general.  So i went with teh approach you had for properties.